file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237
        throw new Error(y, { cause: u });
              ^

Error: [05-May-2025 23:29:09 UTC] PHP Warning:  Undefined array key "REMOTE_ADDR" in /wordpress/wp-content/plugins/limit-access/limit-access.php on line 56
[05-May-2025 23:29:09 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT * FROM wp_limit_access WHERE user_id = 1</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT * FROM `wp_limit_access` WHERE `user_id` = 1 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 no such table: wp_limit_access
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('SELECT * FROM w...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('SELECT * FROM w...')
#3 /wordpress/wp-content/plugins/limit-access/limit-access.php(62): WP_SQLite_DB->query('SELECT * FROM w...', 'ARRAY_N')
#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('limit-access/li...')
#6 /internal/eval.php(20): activate_plugin('limit-access/li...')
#7 {main}</pre>
 for query SELECT * FROM wp_limit_access WHERE user_id = 1 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/limit-access/limit-access.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error
[05-May-2025 23:29:09 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>
		DELETE FROM wp_limit_access
		WHERE (user_ip = '' AND user_id = 1)
		OR timestamp < DATE_SUB(CURRENT_TIMESTAMP, INTERVAL 60 SECOND)
	</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DELETE FROM `wp_limit_access` WHERE ( `user_ip` = '' AND `user_id` = 1 ) OR `timestamp` &lt; DATETIME(CURRENT_TIMESTAMP, '-' || 60 || ' SECOND') | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 no such table: wp_limit_access
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('\n\t\tDELETE FROM ...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('\n\t\tDELETE FROM ...')
#3 /wordpress/wp-content/plugins/limit-access/limit-access.php(78): WP_SQLite_DB->query('\n\t\tDELETE FROM ...')
#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('limit-access/li...')
#6 /internal/eval.php(20): activate_plugin('limit-access/li...')
#7 {main}</pre>
 for query 
		DELETE FROM wp_limit_access
		WHERE (user_ip = '' AND user_id = 1)
		OR timestamp < DATE_SUB(CURRENT_TIMESTAMP, INTERVAL 60 SECOND)
	 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/limit-access/limit-access.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error
[05-May-2025 23:29:09 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_limit_access` (`user_id`, `user_ip`) VALUES (1, '')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT 1 FROM sqlite_temp_schema WHERE type = 'table' AND name = ? | parameters: wp_limit_access</li>
<li>Executing: 
				SELECT column_name, is_nullable, column_default, data_type, extra
				FROM `_wp_sqlite_mysql_information_schema_columns`
				WHERE table_schema = ?
				AND table_name = ?
				ORDER BY ordinal_position
			 | parameters: database_name_here, wp_limit_access</li>
<li>Executing: INSERT INTO `wp_limit_access` () SELECT  FROM (VALUES ( 1 , '' )) WHERE true | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 near ")": syntax error
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('INSERT INTO `wp...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('INSERT INTO `wp...')
#3 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#4 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_limit_access', Array, NULL, 'INSERT')
#5 /wordpress/wp-content/plugins/limit-access/limit-access.php(86): wpdb->insert('wp_limit_access', Array)
#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('limit-access/li...')
#8 /internal/eval.php(20): activate_plugin('limit-access/li...')
#9 {main}</pre>
 for query INSERT INTO `wp_limit_access` (`user_id`, `user_ip`) VALUES (1, '') made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/limit-access/limit-access.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error
[05-May-2025 23:29:18 UTC] PHP Warning:  Undefined array key "REMOTE_ADDR" in /wordpress/wp-content/plugins/limit-access/limit-access.php on line 56
[05-May-2025 23:29:18 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT * FROM wp_limit_access WHERE user_id = 1</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT * FROM wp_limit_access WHERE user_id = 1 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_limit_access.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/limit-access/limit-access.php(62): WP_SQLite_DB->query('SELECT * FROM w...', 'ARRAY_N')
#2 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#3 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('limit-access/li...')
#4 /internal/eval.php(20): activate_plugin('limit-access/li...')
#5 {main}</pre>
 for query SELECT * FROM wp_limit_access WHERE user_id = 1 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/limit-access/limit-access.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error
[05-May-2025 23:29:18 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>
		DELETE FROM wp_limit_access
		WHERE (user_ip = '' AND user_id = 1)
		OR timestamp < DATE_SUB(CURRENT_TIMESTAMP, INTERVAL 60 SECOND)
	</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: 
		DELETE FROM wp_limit_access
		WHERE (user_ip = :param0  AND user_id = 1)
		OR timestamp &lt; DATETIME(CURRENT_TIMESTAMP,   '-60 SECOND')
	 | parameters: </li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_limit_access.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/limit-access/limit-access.php(78): WP_SQLite_DB->query('\n\t\tDELETE FROM ...')
#2 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#3 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('limit-access/li...')
#4 /internal/eval.php(20): activate_plugin('limit-access/li...')
#5 {main}</pre>
 for query 
		DELETE FROM wp_limit_access
		WHERE (user_ip = '' AND user_id = 1)
		OR timestamp < DATE_SUB(CURRENT_TIMESTAMP, INTERVAL 60 SECOND)
	 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/limit-access/limit-access.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error
[05-May-2025 23:29:18 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_limit_access` (`user_id`, `user_ip`) VALUES (1, '')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_limit_access` (`user_id`, `user_ip`) VALUES (1, :param0 ) | parameters: </li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_limit_access.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_limit_access', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/limit-access/limit-access.php(86): wpdb->insert('wp_limit_access', Array)
#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('limit-access/li...')
#6 /internal/eval.php(20): activate_plugin('limit-access/li...')
#7 {main}</pre>
 for query INSERT INTO `wp_limit_access` (`user_id`, `user_ip`) VALUES (1, '') made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/limit-access/limit-access.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error
[05-May-2025 23:29:27 UTC] PHP Warning:  Undefined array key "REMOTE_ADDR" in /wordpress/wp-content/plugins/limit-access/limit-access.php on line 56
[05-May-2025 23:29:27 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT * FROM wp_limit_access WHERE user_id = 1</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT * FROM wp_limit_access WHERE user_id = 1 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_limit_access.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/limit-access/limit-access.php(62): WP_SQLite_DB->query('SELECT * FROM w...', 'ARRAY_N')
#2 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#3 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('limit-access/li...')
#4 /internal/eval.php(20): activate_plugin('limit-access/li...')
#5 {main}</pre>
 for query SELECT * FROM wp_limit_access WHERE user_id = 1 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/limit-access/limit-access.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error
[05-May-2025 23:29:27 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>
		DELETE FROM wp_limit_access
		WHERE (user_ip = '' AND user_id = 1)
		OR timestamp < DATE_SUB(CURRENT_TIMESTAMP, INTERVAL 60 SECOND)
	</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: 
		DELETE FROM wp_limit_access
		WHERE (user_ip = :param0  AND user_id = 1)
		OR timestamp &lt; DATETIME(CURRENT_TIMESTAMP,   '-60 SECOND')
	 | parameters: </li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_limit_access.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/limit-access/limit-access.php(78): WP_SQLite_DB->query('\n\t\tDELETE FROM ...')
#2 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#3 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('limit-access/li...')
#4 /internal/eval.php(20): activate_plugin('limit-access/li...')
#5 {main}</pre>
 for query 
		DELETE FROM wp_limit_access
		WHERE (user_ip = '' AND user_id = 1)
		OR timestamp < DATE_SUB(CURRENT_TIMESTAMP, INTERVAL 60 SECOND)
	 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/limit-access/limit-access.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error
[05-May-2025 23:29:27 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_limit_access` (`user_id`, `user_ip`) VALUES (1, '')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_limit_access` (`user_id`, `user_ip`) VALUES (1, :param0 ) | parameters: </li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_limit_access.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_limit_access', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/limit-access/limit-access.php(86): wpdb->insert('wp_limit_access', Array)
#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('limit-access/li...')
#6 /internal/eval.php(20): activate_plugin('limit-access/li...')
#7 {main}</pre>
 for query INSERT INTO `wp_limit_access` (`user_id`, `user_ip`) VALUES (1, '') made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/limit-access/limit-access.php'), WP_SQLite_DB->query, WP_SQLite_DB->print_error
[05-May-2025 23:31:51 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/lauro-socializer/lauro-socializer.php:181
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('lauro-socialize...')
#2 /internal/eval.php(20): activate_plugin('lauro-socialize...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/lauro-socializer/lauro-socializer.php on line 181
[05-May-2025 23:31:59 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/lauro-socializer/lauro-socializer.php:181
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('lauro-socialize...')
#2 /internal/eval.php(20): activate_plugin('lauro-socialize...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/lauro-socializer/lauro-socializer.php on line 181
[05-May-2025 23:32:06 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/lauro-socializer/lauro-socializer.php:181
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('lauro-socialize...')
#2 /internal/eval.php(20): activate_plugin('lauro-socialize...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/lauro-socializer/lauro-socializer.php on line 181

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
  [cause]: Error: Error when executing the blueprint step #5 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"layout-the-content"},"options":{"activate":true}}) : Could not download "https://downloads.wordpress.org/plugin/layout-the-content.latest-stable.zip".
  				Check if the URL is correct and the server is reachable.
  				If it is reachable, the server might be blocking the request.
  				Check the browser console and network tabs for more information.
  
  				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
  
  				This means the server that hosts your file does not allow requests from other sites
  				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
  				cross-origin file downloads. Learn more about CORS at
  				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
  
  				If your file is on GitHub, load it from "raw.githubusercontent.com".
  				Here's how to do that:
  
  				1. Start with the original GitHub URL of the file. For example:
  				https://github.com/username/repository/blob/branch/filename.
  				2. Replace "github.com" with "raw.githubusercontent.com".
  				3. Remove the "/blob/" part of the URL.
  
  				The resulting URL should look like this:
  				https://raw.githubusercontent.com/username/repository/branch/filename
  
  				Error:
  				Error: Could not download "https://downloads.wordpress.org/plugin/layout-the-content.latest-stable.zip"
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)
      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
    [cause]: Error: Could not download "https://downloads.wordpress.org/plugin/layout-the-content.latest-stable.zip".
    				Check if the URL is correct and the server is reachable.
    				If it is reachable, the server might be blocking the request.
    				Check the browser console and network tabs for more information.
    
    				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
    
    				This means the server that hosts your file does not allow requests from other sites
    				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
    				cross-origin file downloads. Learn more about CORS at
    				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
    
    				If your file is on GitHub, load it from "raw.githubusercontent.com".
    				Here's how to do that:
    
    				1. Start with the original GitHub URL of the file. For example:
    				https://github.com/username/repository/blob/branch/filename.
    				2. Replace "github.com" with "raw.githubusercontent.com".
    				3. Remove the "/blob/" part of the URL.
    
    				The resulting URL should look like this:
    				https://raw.githubusercontent.com/username/repository/branch/filename
    
    				Error:
    				Error: Could not download "https://downloads.wordpress.org/plugin/layout-the-content.latest-stable.zip"
        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)
  }
}

Node.js v23.11.0
